src/renderer/component/publishForm/view.jsx

```diff
@@ -526,6 +582,14 @@ class PublishForm extends React.PureComponent {
     });
   }
 
+  upload(path) {
+    this.closeModal();
+    this.setState({
+      thumbnailUploadStatus: "sending",
```

Comment:
sending status would be better in redux than local state... but it does make sense that you would have it like this for quickly developing the other parts